### PR TITLE
Feat : adding header compoent

### PR DESCRIPTION
**Pull request by Pratik33**
Feat :
1. adding header component
2. code with React.create element and JSX

### CODE REVIEW

File: 03-functional-component/src/App.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import ReactDOM from 'react-dom/client';
+import React from 'react';
+import './index.css';
```
Comment (ap221882) [inline 03-functional-component/src/App.js:3]: Leave a blank line between global imports and imports and the other code. Improves readability.

File: 03-functional-component/src/index.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
+:root {
```
Comment (ap221882) [inline 03-functional-component/src/index.css:1]: First assignment in which I found someone using CSS variables. Kudos!

File: 03-functional-component/src/index.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  </head>
+
+  <body>
+    <div id="root"></div>
+    <script
+      type="module"
+      src="App.js"></script>
+    <script
+      src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
+      integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo"
```
Comment (ap221882) [inline 03-functional-component/src/index.html:24]: ES modules are deferred by default and you can use defer with other scripts too to move them to head component. It just scripts look good at head tag. Also use prettier for uniform formatting.

File: 03-functional-component/src/App.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+  <div className='title'>
+    {' '}
+    <h1>React Bootcamp - JSX</h1>
+    <h2>By Pratik Mothe</h2>
+    <h4>Assignment number - {assignmentNumber}</h4>
+  </div>
+);
+
+// Functional components
+// A javascript function returning a valid JSX / react element is call component.
```
Comment (ap221882) [inline 03-functional-component/src/App.js:23]: Try multiline comments with better comments extension. They are more readable.

File: 03-functional-component/src/index.html [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+      crossorigin="anonymous"></script>
+
+    <script
+      src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.0/umd/popper.min.js"
+      integrity="sha384-cs/chFZiN24E4KMATLdqdvsezGxaGsi4hLGOzlXwp5UZB1LY//20VyM2taTB4QvJ"
+      crossorigin="anonymous"></script>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
```
Comment (ap221882) [inline 03-functional-component/src/index.html:33]: Put the stylesheets in head. It is good that your styles load before the HTML parses.

**Review by ap221882 — COMMENTED** [review]
Nicely written. Appreciate the semantic HTML and CSS you tried. Keep it up!

**Comment by Pratik33** [comment]
@ap221882 @upaharika @ruchamahabal thanks for reviewing the code.
few changes are made in assignment 4

**Pull request merged**